map.go

```diff
@@ -106,6 +118,22 @@ func FromJSON(jsonString string) (Map, error) {
 	return m, nil
 }
 
+// FromJSONSlice creates a new slice of Map containing the data specified in the
+// jsonString. Works with jsons with a top level array
+//
+// Returns an error if the JSON is invalid.
+func FromJSONSlice(jsonString string) ([]Map, error) {
+	var slice []Map
+	err := json.Unmarshal([]byte(jsonString), &slice)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return Nil, err
```

Reply:
This won't build, you can return `nil` or `[]Map{}` (or `slice` here, but this might be confusing) here. I've no strong preference :shrug:

Reply:
Oh right! `nil` it is then.